inductiva/molecules/protein_solvation/README.md

```diff
@@ -2,14 +2,14 @@
 
 This scenario models the dynamics of a protein solvated in water inside a cubic box, powered by the GROMACs simulator. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
 
+To access all the visualization features of this scenario, you need to install the extra dependencies for the molecules package with `pip install --upgrade "inductiva[molecules_extra]"`. 
+
 To initialize the scenario, the user can define the following parameters:
 - `protein_pdb` : PDB file containing the initial conformation of the protein;
 - `temperature`: Temperature of the system in K.
 
-Before simulation, files undergo automatic pre-processing in our machines to strip away all non-protein components of the system, such 
-as ligands, non-standard residues, or water molecules, leaving only the protein.
-We tested the scenario with structures predicted by the [AlphaFold2 model](https://alphafold.ebi.ac.uk) and structures from the [RCSB Protein Data Bank](https://www.rcsb.org), but you can provide any other well-formatted PDB file as input, as long as its contains at least one protein chain with all its atoms coordinates. 
-
+Before simulation, files undergo automatic pre-processing in our machines to strip away all non-protein components of the system, such as ligands, non-standard residues, or water molecules, leaving only the protein.
+We tested the scenario with structures predicted by the [AlphaFold2 model](https://alphafold.ebi.ac.uk) and structures from the [RCSB Protein Data Bank](https://www.rcsb.org), but you can provide any other well-formatted PDB file as input, as long as its contains at least one protein chain with all its atoms coordinates. Please be aware that PDB files downloaded directly from RCSB may need extra preparation, like filling in missing atoms. Our API doesn't handle this kind of preparation, so you can use software like [Modeller](https://salilab.org/modeller/) or [Chimera](https://www.cgl.ucsf.edu/chimera/) for this purpose.
```

Comment:
I don't really like that we are saying this in the end... Since we actually do some pre-processing to strip things out and then for other extra work, like putting things in, we don't do it.

This bring any errors to the users? Have you found any?

Reply:
Yes, it is a big problem with the RCSB PDB files. I opened an issue, about it actually. Look here to understand better https://github.com/inductiva/inductiva-web-api/issues/311. But I changed the text to: "Please be aware that PDB files downloaded directly from RCSB may need extra preparation, other than removing non-protein elements, like filling in missing atoms. Our API doesn't handle this of pre-processing step, so you can use software like [Modeller](https://salilab.org/modeller/) or [Chimera](https://www.cgl.ucsf.edu/chimera/) for this purpose."